unblob/handlers/compression/compress.py

```diff
@@ -48,7 +48,9 @@ class UnixCompressHandler(StructHandler):
             // magic followed by valid flags bytes (!(flag & 0x60) && (9 <= (flag & 0x1F) <= 16))
             $compress_magic = { 1f 9d ( 09 | 0A | 0B | 0C | 0D | 0E | 0F | 10 | 89 | 8A | 8B | 8C | 8D | 8E | 8F | 90 ) }
         condition:
-            $compress_magic
+            $compress_magic and
+            // first code must be a literal
```

Comment:
A bit more context would be helpful here, what does it mean that it must be a literal?

Reply:
Added link to the compress format reference and explained the different kinds of codes in LZW.